### PR TITLE
Fix typos in tests/

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: tests/zfs-tests/tests/functional/cli_root/zpool_create/zpool_create_016_pos.ksh [hunk 1]
```diff
@@ -41,7 +41,7 @@
 # STRATEGY:
 # 1. delete all devices in the swap
 # 2. create a zpool
-# 3. Verify the creation is successed.
+# 3. Verify the creation is succeed.
```
[the remark this answers is not in the document]
Comment (Gelma) [reply tests/zfs-tests/tests/functional/cli_root/zpool_create/zpool_create_016_pos.ksh]: Great I like the second option.

I commit and squash!

File: tests/zfs-tests/tests/functional/cli_root/zpool_create/zpool_create_016_pos.ksh [hunk 2]
```diff
@@ -41,7 +41,7 @@
 # STRATEGY:
 # 1. delete all devices in the swap
 # 2. create a zpool
-# 3. Verify the creation is successed.
+# 3. Verify the creation was successful
```
Comment (rlaager) [inline tests/zfs-tests/tests/functional/cli_root/zpool_create/zpool_create_016_pos.ksh]: This needs a period at the end.

Comment (Gelma) [reply tests/zfs-tests/tests/functional/cli_root/zpool_create/zpool_create_016_pos.ksh]: Done!

Comment (Gelma) [reply tests/zfs-tests/tests/functional/cli_root/zpool_create/zpool_create_016_pos.ksh]: Done.

Comment (rlaager) [reply tests/zfs-tests/tests/functional/cli_root/zpool_create/zpool_create_016_pos.ksh]: The others are still inconsistent, but meh.

Comment (Gelma) [reply tests/zfs-tests/tests/functional/cli_root/zpool_create/zpool_create_016_pos.ksh]: > The others are still inconsistent, but meh.

Uploaded version with full stop removd

**Pull request merged**